structure/0006-governance.md

```diff
@@ -2,53 +2,128 @@
+Sub teams behave like regular teams, except they are not required to have a team lead.
+If the team doesn't have its own team lead, the parent's team leads act as its team lead.
+In general, a sub team inherits its parent's structure according to their parent's team RFC.
 
-#### Explanation
-What will this team *do*? What new projects will be created that this team will be responsible for and how will each of these projects benefit the QuiltMC organization/community? What processes will this team follow for each of the different kinds of changes their projects may undergo?
+## The Administrative Board
 
-#### Drawbacks
-Can usually be omitted. Is only necessary if there are clear drawbacks to such a teams existence. Drawbacks may include overlap of responsibilities, excessive granularity, etc.
+The administrative board is a group of organization members responsible for tasks that are not assigned to another team.
+It is made up of at least three members to prevent deadlocks in the decision-making process.
+Members of the administrative board may be members of other teams.
 
-#### Rationale and Alternatives
-Can be omitted. Who is currently responsible for existing projects this team will takeover? What teams might become responsible for new projects proposed in this document if the team itself is deemed unnecessary?
+At any point in time, a member of the organization may nominate themselves for a position on the administrative board.
+This will then initiate the voting process, with all other organization members being eligible to vote.
+If the nominated member passes a two-thirds vote, they are added to the administrative board.
 
-#### Prior Art
-Can usually be omitted. Is only necessary if there are prior organizations/projects with very similar goals.
+If there are less than three members on the administrative board, a special vote is initiated.
+Each remaining board member may nominate an organization member.
+Organization members may also nominate themselves.
+All other organization members may then vote on the nominees.
+The nominee with the most votes will be added to the administrative board.
+In the case of a tie, the remaining board members must break that tie.
+
+## Processes
+
+To streamline working with teams, some standard processes need to be defined.
+The following processes apply to all teams unless that team's RFC specifies a different process.
+
+### Voting
+
+Unless specified otherwise, the following process is used for voting:
+
+A vote in a team may be called at any time by a team lead for that team or for one of its parent teams.
+A member of the administrative board may also initiate a vote in any team. 
+
+When a vote is called within a team, all members of that team, its parent teams, and the administrative board are eligible to vote.
+All polls should include a positive and negative option, as well as the option to abstain.
+Voting members that abstain are excluded from percentages and tallies.
```

Comment:
How does this handle voting members who did not vote within the allotted time frame? Are they counted as having abstained, or separately marked as "not voting"?

Reply:
Right now I believe we count them as abstained, I think that's a good default option